authentication/urls.py

```
@@ -0,0 +1,27 @@
+from django.urls import path
+from django.urls import include
```

Ruff (0.8.2)

`django.urls.include` imported but unused

Remove unused import: `django.urls.include`

(F401)